frontend/src/style.css

```
@@ -2588,3 +2588,100 @@ a:hover { color: var(--primary-hover); }
 .analytics-error p {
   margin-bottom: 0.5rem;
 }
+
+/* === Completion Feedback === */
+.completion-feedback-overlay {
+  position: fixed;
+  inset: 0;
+  background: rgba(0,0,0,0.5);
```

Stylelint (17.2.0)

[error] Expected "0.5" to be "50%" (alpha-value-notation)

(alpha-value-notation)